book/007-Modern-JS-Architecture.md

```diff
@@ -99,21 +99,21 @@ Personally speaking here, **I find learning how to configure build systems extre
 
 For quick and easy setups with common components and pieces like `React` and `TypeScript`, I recommend using `Parcel`.
 
-To my knowledge, `Parcel` as a configuration-free build system, as `Webpack` is complicated and difficult to set up, especially for beginners. **Webpack can be intimidating and difficult for newcomers**.
+To my knowledge, `Parcel` is a configuration-free build system and  `Webpack` is complicated and difficult to set up, especially for beginners. **Webpack can be intimidating and difficult for newcomers**.
```

Comment:
Note to self... Don't say webpack is complicated and difficult to set up. There's got to be a better way to phrase this.
```suggestion
To my knowledge, `Parcel` is a configuration-free build system and  `Webpack` has a tendency to be significantly more complicated and difficult to set up. For those just learning JavaScript, and don't have as much hands-on experience, **Webpack can be intimidating and difficult**.
```